### DISCUSSION TITLE
Please reconsider support for Mojave (last OSX supporting 32bit applications)

### DISCUSSION THREAD

**Discussion by karlacio** [Homebrew/discussions#2450] [Everyday usage]
Hi,

I know that brew support only the last 3 versions of macOS and that supporting an extra version is not easy.

MacOS 10.14 Mojave is the latest OS from Apple that support 32bit applications. Some of us are tied to this OS due to legacy applications.

It would be great to have access to all software from brew without having to recompile and troubleshoot errors during updates.

I don't know how many are tied to this old version.
Maybe we are a significant number to justify this extra support.

Thanks,
Carlo.

**Comment by carlocab** [comment]
Less than 10% of Homebrew users are still on Mojave: https://formulae.brew.sh/analytics/os-version/365d/

We already maintain six different bottle builds for each formula; I don't think we can afford to invest the extra resources to maintain another one that will be used by only a small fraction of users.

**Reply by andig** [reply]
The table is very odd. Catalina has double the events than big sur but half the percentage?

**Reply by carlocab** [reply]
Yep, that doesn't seem right. CC @MikeMcQuaid since you looked at this recently.

**Reply by SMillerDev** [reply]
That might be because macOS runners on Github are on Catalina by default?

**Reply by carlocab** [reply]
That might be true, but that still doesn’t explain why the raw numbers don’t seem to match the percentages.

**Reply by MikeMcQuaid** [reply]
Yeh, this is likely from https://github.com/Homebrew/homebrew-formula-analytics/pull/277. If any of y'all can spot the bug there I'd be grateful.

https://formulae.brew.sh/analytics/os-version/30d/ is more accurate and more stark.

**Comment by tjluoma** [comment]
MacPorts has installers for Mojave. It's what I'm using for my older Macs. https://www.macports.org
